docs/resources/hvn_route.md

```diff
@@ -0,0 +1,89 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_hvn_route Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The HVN route resource allows you to manage an HVN route.
+---
+
+# hcp_hvn_route (Resource)
+
+The HVN route resource allows you to manage an HVN route.
+
+## Example Usage
+
+```terraform
+provider "aws" {
+  region = "us-west-2"
+}
+
+resource "hcp_hvn" "main" {
+  hvn_id         = "main-hvn"
+  cloud_provider = "aws"
+  region         = "us-west-2"
+  cidr_block     = "172.25.16.0/20"
+}
+
+// Creating a peering and a route for it.
+resource "aws_vpc" "peer" {
+  cidr_block = "192.168.0.0/20"
+}
+
+resource "hcp_aws_network_peering" "example" {
+  peering_id      = "peer-example"
+  hvn_id          = hcp_hvn.main.hvn_id
+  peer_vpc_id     = aws_vpc.peer.id
+  peer_account_id = aws_vpc.peer.owner_id
+  peer_vpc_region = "us-west-2"
+}
+
+resource "aws_vpc_peering_connection_accepter" "peer" {
+  vpc_peering_connection_id = hcp_aws_network_peering.example.provider_peering_id
+  auto_accept               = true
+}
+
+resource "hcp_hvn_route" "example-peering-route" {
+  hvn              = hcp_hvn.main.self_link
+  destination_cidr = aws_vpc.peer.cidr_block
+  hvn_route_id     = "peering-route"
+  target_link      = hcp_aws_network_peering.example.self_link
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **destination_cidr** (String) The destination CIDR of the HVN route
+- **hvn** (String) The `self_link` of the HashiCorp Virtual Network (HVN).
+- **hvn_route_id** (String) The ID of the HVN route.
+- **target_link** (String) A unique URL identifying the target of the HVN route. Examples of the target: [`aws_network_peering`](aws_network_peering.md), [`aws_transit_gateway_attachment`](aws_transit_gateway_attachment.md)
```

Comment:
@bcmdarroch I wonder if you know if these links would work that way in the TF registry

Reply:
Yes this should work.